### ISSUE TITLE
Exec command in i3/.config file on Regolith 

### ISSUE THREAD

**Issue by ThSGM** [regolith-linux/regolith-desktop#523]
**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: [e.g. 2.90]
 - PPA url: [default: ppa:regolith-linux/release]
 - Host OS (for PPA):  Ubuntu 20.04

**Additional context**
[i3-alternating-layout plugin](https://github.com/olemartinorg/i3-alternating-layout)

I am new to Linux and am struggling a bit with handling exec codes from Regolith. Specifically i am trying to install this [plugin]([i3-alternating-layout plugin](https://github.com/olemartinorg/i3-alternating-layout). The important thing is that I have placed the following command into `~/.config/regolith/i3/config':

```
# alternating-layout python
exec --no-startup-id $HOME/.dotfiles/i3/alternating_layouts.py
```

I've made sure that the file is there. However when I save the above config file and re-run Regolith via super+shift+R, the alternate tyling functionality does not seem to be implemented. 

in contrast, I can manually run the alternating.layouts.py file via python and it seems to run no problem.

Does anyone have any idea how I can try and track down the srouce of the issue with the exec command?

**Comment by kgilmer** [comment]
Hi @ThSGM nice to hear you're working with Linux!  Regarding your issue, I'm guessing that at least $HOME is not expanding to your home directory as you would expect.  The i3 config file is not a shell script, it's a file that's read just by i3, and has it's own syntax and rules.  You could set a variable in i3 to your home dir, or just specify the path absolutely (`/home/me/.dotfiles...`)

Pleaes re-open if this does not resolve your issue!

**Comment by ThSGM** [comment]
I managed to find the issue. The issue was due to my ignorance of how the exec command works. Because the python file is executed without the python command by exec, then we need to make sure we have the proper header on top of the python file. In my case, this means

`#!/usr/bin/python3`

should be placed at the start of the python script. 

I am not sure but I may have had to log out and back in to ensure this was working after the change.
